### PR TITLE
chore: prettier, commit msg check

### PR DESCRIPTION
**Pull request by qndrey**
What I did:

1. Added prettier to be more consistent with code styles
2. Added 'npm run lint:fix' to fix all issues with codestyles
3. Added commit linter to validate a proper commit message (https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional) It will be easier to add an automatic generated changelog when release a tag/version of the website.

### CODE REVIEW

**Comment by f10l** [comment]
@yash296 can you perform a review here?

@andrewBalekha It seems that the develop branch is not up to date, sorry about that :-/ 

Easy workaround, would be to perform this PR against master (currently, the git flow is not really implemented for this repo). Afterwards we could easier update the develop branch.

File: src/store/index.js [hunk 1]
```diff
@@ -1,15 +1,11 @@
-import Vue from 'vue'
-import Vuex from 'vuex'
+import Vue from "vue";
+import Vuex from "vuex";
```
Comment (qndrey) [inline src/store/index.js:2]: I can remove `;` if you guys are not using them,

**Review by qndrey — COMMENTED** [review]
I also fixed two issues:
Screenshot 2020-04-27 at 09 35 53

Animation and left align, thus it looks better in my opinion.
Screenshot 2020-04-27 at 09 36 35

**Comment by qndrey** [comment]
Also, I would not use underline style for headlines, it feels like a link but it is not.

**Pull request merged**